### DISCUSSION TITLE
"Prompt-to-Prompt Image Editing with Cross Attention Control"

### DISCUSSION THREAD

**Discussion by lendrick** [hlky/stable-diffusion-webui#939] [Feature suggestions]
Saw this in a thread posted on reddit the other day:

https://www.reddit.com/r/StableDiffusion/comments/x98py5/reproducing_the_method_in_prompttoprompt_image/

I recommend checking out the paper.  Apparently some really amazing stuff is possible with this.

https://arxiv.org/abs/2208.01626

There's someone working on implementing it.  I'm not sure how far they've gotten or if they have a github repo yet, but this is definitely something worth keeping an eye on.

**Comment by 831Digital** [comment]
It has an implementation here: https://github.com/bloc97/CrossAttentionControl

Would love to see this added.

**Reply by mchaker** [reply]
(bumping, upvoting, thumbs up with both hands and feet)

**Comment by HalfMAI** [comment]
Try using this commit
https://github.com/Doggettx/stable-diffusion/commit/ccb17b55f2e7acbd1a112b55fb8f8415b4862521

Result:
![NKSX 3R0Y @%1OUZI)7OQEN](https://user-images.githubusercontent.com/5293796/191047003-5c1eb996-bed9-4caa-9c0c-7dec00572316.png)
![E7U5X~C(CM4OO3ENAF7DC_5](https://user-images.githubusercontent.com/5293796/191047076-1a19f3ef-5f43-4805-b970-765b8c713036.png)
![@`K}NDRS _DI}E}E82$BHVX](https://user-images.githubusercontent.com/5293796/191047645-aadc2860-e0e8-4d82-a5d9-4fd802cbc331.jpg)


![2GU@L3F1LH 5VI PZ~U4)X2](https://user-images.githubusercontent.com/5293796/191047538-58ca7651-27ec-4a3f-a4a6-188d9b134c12.jpg)
![Z {K)LAA8ODHGW85LF@F~RS](https://user-images.githubusercontent.com/5293796/191047556-59b76d5d-a074-4068-8b58-d4fa769addd9.jpg)

**Comment by HalfMAI** [comment]
Made a pull request(https://github.com/sd-webui/stable-diffusion-webui/pull/1229)
But I'm very new to this repo, hope someone more familiar can help
